### ISSUE TITLE
Deathstalker Erland spams various warnings whether on his escort quest or not

### ISSUE THREAD

**Issue by wirerydr** [elysium-project/bug-tracker#287]
**Server**: Nighthaven PvP
**Patch**: 1.4
**Client**: 1.12.1
**Time/Date**: 2018-01-08, Approx 06:05am server-time
## Synopsis
Deathstalker Erland spams attack-warnings multiple times per second
## Description
Deathstalker Erland at/near Ivar's Patch will spam multiple warnings per second (looks like about 6 msg/sec) if he comes under attack by a mob such as a Ravenclaw Slave.  This was observed happening both while I was personally escorting him to safety, and also while he was standing (apparently unattended) beside Rane Yorick in front of his cabin.

See the attached screenshot for an illustration.  The screen shot shows about 4-5 seconds of the spam, but my chat-log continuously populated with this until the Ravenclaw Slave was killed.

![image](https://user-images.githubusercontent.com/4572687/34659674-74bf55f8-f409-11e7-969b-1700943374a1.png)

**Comment by Terranzin** [comment]
I´ve tried to reproduce the issue you´ve reported. You can see in my picture the whole chatlog when I did the escort (and I intentionally aggroed Ravenclaw Slave). I can´t seem to be able to reproduce it on our PTR but since you have a picture as proof there seems to be differences in that case.

![image](https://user-images.githubusercontent.com/25831382/34844103-ab0a3e02-f710-11e7-979f-d31d53b27761.png)

**Comment by wirerydr** [comment]
I can no longer reproduce the issue.  After reading your experience I ran the escort quest again with a different toon, and then waited around to observe a separate group escort him.  In both instances Erland appeared to only say his warnings once per mob.  I made only one substantial chat-related addon change (see below), and no changes at-all to my UI interface settings.

I DID upgrade my spam-throttle to a newer version (about 14 months newer), so on the off-chance that has an impact I will try to recover the original version and reattempt my tests, and report back.
